main.ts

```diff
@@ -257,13 +257,14 @@ export default class MyPlugin extends Plugin {
 				const end = e.position.end;
 				myLine.formatted = myLine.formatted.replace(line.substring(start.col, end.col), e.displayText);
 			});
+
+			// Regex of all the supported alternate checkbox styles
+			const cbRe = /^- \[[ x\/\-><?!*\"lbiSpcfkwud]\]/gi;
```

Comment:
Actually marks every single character except whitespace, a checkbox, as done. Other themes might have additional checkbox styles as well. So I think the regex should be widened to all characters.

Reply:
Updated the regex pattern to match against any character between the `[ ]`

Reply:
A `- [ ]` line is replaced as well. I think whitespaces shouldn't be replaced, because they are not marked as done.

Reply:
Oh, OK, I was thinking to apply it to all types of checkboxes, done and not done.
I've updated the regex pattern to not match against checkboxes not marked as done, i.e. "- [ ]"